### PR TITLE
Update API documentation

### PR DESCRIPTION
**Pull request by atallahade**

### CODE REVIEW

File: src/docs/content/docs/api.md [hunk 1]
```diff
@@ -58,18 +58,19 @@ This operation expects that a call was previously made to `/login` and that the
...
   * Expected query parameters:
       * *seedHost*: New host name or IP address used as Cassandra cluster seed.
+      * *jmxPort*: Custom JMX port.
   * Modifies a cluster's seed host. Comes in handy when the previous seed has left the cluster.
   
   
 * **DELETE  /cluster/{cluster_name}**
-  * Expected query parameters:
-      * *force* : Enforce deletion of the cluster even if there are active schedules and a repair run history (Optional)
+  * Expected query parameters: *None*
```
Comment (adejanovski) [inline src/docs/content/docs/api.md]: You might need to rebase on top of master to get the `force` param back. It's a new addition from Reaper 2.0.

**Review by adejanovski — CHANGES_REQUESTED** [review]
Thanks for updating the docs @atallahade. 

Your PR removes the `force` param from the cluster deletion call. Could you bring it back before we merge?

**Comment by atallahade** [comment]
@adejanovski I've also documented a missing endpoint.

**Comment by adejanovski** [comment]
Hi @atallahade,

sorry I forgot to merge this PR. There are merge conflicts that need to be addressed, and the commits need to be squashed before I can merge it.

Thanks!

**Pull request merged**